mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/impl/MongoDBDatastoreProvider.java

```diff
@@ -136,9 +133,6 @@ protected MongoClient createMongoClient(MongoDBConfiguration config) {
 					? new MongoClient( serverAddresses, clientOptions )
 					: new MongoClient( serverAddresses, credentials, clientOptions );
 		}
-		catch (UnknownHostException e) {
-			throw log.mongoOnUnknownHost( config.toString() );
```

Comment:
We don't have any written policy, but removing unused stuff seems good to me. Just not should re-use ids for new messages.